README.md

```diff
@@ -62,74 +64,67 @@ support.
 ### How to install
 
 ```sh
-sudo apt-get install python3-venv
-
-pyvenv .venv
-. .venv/bin/activate
-
-# Stable version
-
-pip install homu
-
-# Development version
-
-git clone https://github.com/barosl/homu.git
-pip install -e homu
+$ git clone https://github.com/barosl/homu.git
+$ pip install -e homu
```

Comment:
I've never used venv so I'm not confident I can write docs explaining to people how they should use it.

Could that be done as a separate PR though? The current PR only reformats the code that's already there, no?